cpp/kiss_icp/core/VoxelHashMap.hpp

```diff
@@ -46,14 +46,18 @@ struct VoxelHashMap {
     struct VoxelHash {
         size_t operator()(const Voxel &voxel) const {
             const uint32_t *vec = reinterpret_cast<const uint32_t *>(voxel.data());
-            return ((1 << 20) - 1) & (vec[0] * 73856093 ^ vec[1] * 19349669 ^ vec[2] * 83492791);
+            return (vec[0] * 73856093 ^ vec[1] * 19349669 ^ vec[2] * 83492791);
         }
     };
 
     explicit VoxelHashMap(double voxel_size, double max_distance, int max_points_per_voxel)
         : voxel_size_(voxel_size),
           max_distance_(max_distance),
-          max_points_per_voxel_(max_points_per_voxel) {}
+          max_points_per_voxel_(max_points_per_voxel),
```

Comment:
Is it too much to ask to split this into a separate PR? This is to avoid chasing ghosts in the future. If we merge this it will appear in the commit history as ``Remove redundant Modulus operation for Voxel Hash function

I can also do it if you give me access to your fork

Reply:
I just did it, some parts where kind of magic numbers there so I decide to keep just the explainable ones in the new PR.